packages/app/src/modules/options/areaConfig.js

```diff
@@ -55,16 +54,20 @@ export default [
         label: i18n.t('Axes'),
         content: [
             {
-                key: 'axes-section-1',
+                key: 'axes-vertical-axis',
+                label: i18n.t('Vertical (y) axis'),
```

Comment:
Maybe we can centralise these labels in a `const` somehow so we don't have to repeat them everywhere.